src/lib/Tests/EventListener/SystemInfoTabGroupListenerTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Tests\EventListener;
+
+use EzSystems\EzPlatformAdminUi\EventListener\SystemInfoTabGroupListener;
+use EzSystems\EzPlatformAdminUi\Tab\SystemInfo\SystemInfoTab;
+use EzSystems\EzPlatformAdminUi\Tab\TabGroup;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use PHPUnit\Framework\TestCase;
+use EzSystems\EzPlatformAdminUi\Tab\Event\TabEvents;
+use EzSystems\EzPlatformAdminUi\Tab\Event\TabGroupEvent;
+use EzSystems\EzPlatformAdminUi\Tab\SystemInfo\TabFactory;
+use EzSystems\EzPlatformAdminUi\Tab\TabRegistry;
+use EzSystems\EzSupportToolsBundle\SystemInfo\SystemInfoCollectorRegistry;
+use PHPUnit_Framework_MockObject_MockObject;
+
+class SystemInfoTabGroupListenerTest extends TestCase
+{
+    /** @var Request */
+    private $request;
+
+    /** @var TabGroupEvent */
+    private $event;
+
+    /** @var HttpKernelInterface|PHPUnit_Framework_MockObject_MockObject */
```

Comment:
Can we use namespaces? I believe we are using PHPUnit 6.x which brought support for these.

Reply:
Of course, I will change it